hadoop-client-modules/hadoop-client-minicluster/pom.xml

```diff
@@ -411,72 +415,20 @@
     <!-- Skip commons-logging:commons-logging-api because it looks like nothing actually included it -->
     <!-- Skip jetty-util because it's in client -->
     <dependency>
-      <groupId>com.sun.jersey</groupId>
-      <artifactId>jersey-core</artifactId>
+      <groupId>org.glassfish.jersey.core</groupId>
```

Comment:
We will use `org.glassfish.jersey.core` to completely replace the original `com.sun.jersey`, so we will completely remove the original dependency from pom.xml.